app/src/main/java/io/intrepid/contest/models/Score.java

```diff
@@ -0,0 +1,28 @@
+package io.intrepid.contest.models;
+
+import java.util.UUID;
+
+public class Score {
+
+    private UUID scoringCategoryId;
+    private int score;
+
+    public Score(UUID scoringCategoryId, int score) {
+        this.scoringCategoryId = scoringCategoryId;
+        this.score = score;
+    }
+
+    UUID getCategoryId() {
+        return scoringCategoryId;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
In the RateEntryPresenterTest I'm using eq(new Score()); There may be a better way to test this though. It was just the most straightforward way I could think of